runtime/src/runtime/policy.rs

```diff
@@ -336,7 +336,7 @@ pub mod policy_constants {
     /// Maximum number of epochs past the current epoch a sector may be set to expire.
```

Comment:
I think a naive implementation of FIP 56 text:

> The minimum sector duration time will increase from 6 months to 1 year and the maximum sector duration will increase from 1.5 years to 5 years. 

Would lead to use increasing this parameter to 1 * EPOCHS_IN_YEAR since this is how pre commit validates sector lifetime.

However I don't think the intent of the FIP is to force minimum expiration extension increments to be one year so I think a better implementation of this is going to require adding another parameter MIN_SECTOR_LIFETIME, setting this to one year and checking this parameter in precommit.

Its unclear enough that it probably warrants consulting with the FIP authors on their intention.

However